doc/tutorial/introduction/introduction_align.cpp

```diff
@@ -0,0 +1,74 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
```

Comment:
Usually we do not put the copyright for snippets.

Reply:
The snippets and tests belong to our library and must be copyrighted as well.
If we have the copyright already missing in other tutorial's snippets, we should add them... especially to explicitly grant our users the right to copy and use the snippet code.
@smehringer what do you think?

Reply:
No more `\include` :(

But while the tests include the copyright, none the snippets contain a copyright.

Reply:
The snippets should not contain this copyright, because it would force anyone copy'n'pasting from the snippets to include our license which is a little too much, I think. 

We should either 
  * prefix every snippet file with
```cpp
// This file is licensed under CreativeCommons - Zero 1.0 Universal / released into the Public Domain.
// You may use this file as wish, including removing this copyright notice.
// This does not affect obligations to include a notice to SeqAn3's copyright when distributing the SeqAn3 library.
```
  * Or not have any note in these files (so they can be `\include`d and instead put above notice into the LICENSE file in our repository root (with "This file is" changed to "Any C++ source code in the `doc` and `test/snippet`-folders is").

Reply:
The legal implications of copy'n'pasting from the manual should also be covered in the first article. In general I would put more emphasis on the "meta" side of things.

E.g. "This is a code snippet. You will see many code snippets in our documentation. Most of them are compilable as-is, but some are only valid in their context, e.g. they depend on other code snippets given before/after the current one or other statements implied by the text. You can copy'n'paste freely from these examples, this implies no copyright-obligations (however distributing SeqAn3 or an application using it does, see "Copyright and Citing").

[I just realised we need an article like that]

Reply:
:+1: